tests/unit/plugins/module_utils/libraries/powerstore_unit_base.py

```diff
@@ -36,14 +36,6 @@ def capture_fail_json_call(self, error_msg, module_mock, module_handler=None, in
             if error_msg not in fj_object.message:
                 raise AssertionError(fj_object.message)
 
-    def capture_fail_json_method(self, error_msg, module_mock, function_name, *args, **kwargs):
```

Comment:
Is this method not used any of Unit test file?

Reply:
i added this for quota.